deployment-config/fraxtal-eth-l1-08-13-24.json

```diff
@@ -0,0 +1,49 @@
+{
+    "protocolAdmin": "0x0000000000417626Ef34D62C4DC189b021603f2F",
+    "boringVault": {
+        "boringVaultSalt": "0x1000000000000000000000000000000000000000000000000000000000000002",
+        "boringVaultName": "Fraxtal Native Yield Nucleus Token",
+        "boringVaultSymbol": "frxtlETH",
+        "address": "0x0000000000000000000000000000000000000000"
+    },
+    "manager": {
+        "managerSalt": "0x2000000000000000000000000000000000000000000000000000000000000002",
+        "address": "0x0000000000000000000000000000000000000000"
+    },
+    "accountant": {
+        "accountantSalt": "0x3000000000000000000000000000000000000000000000000000000000000002",
+        "payoutAddress": "0x0000000000417626Ef34D62C4DC189b021603f2F",
+        "allowedExchangeRateChangeUpper": "10030",
+        "allowedExchangeRateChangeLower": "9980",
+        "minimumUpdateDelayInSeconds": "3600",
+        "managementFee": "0",
+        "address": "0x0000000000000000000000000000000000000000"
+    },
+    "teller": {
+        "tellerSalt": "0x4000000000000000000000000000000000000000000000000000000000000002",
+        "opMessenger": "0x4200000000000000000000000000000000000007",
```

Comment:
Not important since we are using LZ, but this is messenger address for L2 in case it is ever switched to OP